File: skills/update_config.py
import json

with open('workspace/whatsapp_config.json', 'r') as f:
    config = json.load(f)

if '61423475992' not in config['allowedDMs']:
    config['allowedDMs'].append('61423475992')

with open('workspace/whatsapp_config.json', 'w') as f:
    json.dump(config, f, indent=2)

with open('workspace/whatsapp_config.json', 'r') as f:
    print(f.read())

File: skills/update_config.js
const fs = require('fs');
const path = '/Users/vbalaraman/OpenSpider/workspace/whatsapp_config.json';
const cfg = JSON.parse(fs.readFileSync(path, 'utf8'));
if (!cfg.allowedDMs.includes('14164222657')) cfg.allowedDMs.push('14164222657');
fs.writeFileSync(path, JSON.stringify(cfg, null, 2));
console.log(fs.readFileSync(path, 'utf8'));